hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -87,6 +87,7 @@
     {
       name: flash_ctrl_rd_buff_evict
       uvm_test_seq: flash_ctrl_rd_buff_evict_vseq
+      run_opts: ["+en_cov=1"]
```

Comment:
just curious - do we want to collect coverage even when -cov option is off? I think we usually enable coverage in command line.

Reply:
thanks jaedon for the suggestion! @weicaiyang @sriyerg @matutem any thoughts on this?

Reply:
To unblock the progress, I filed an issue for that. #13628